app/src/main/java/com/keylesspalace/tusky/components/account/AccountActivity.kt

```diff
@@ -890,6 +930,16 @@ class AccountActivity : BottomSheetActivity(), ActionButtonActivity, HasAndroidI
         } else null
     }
 
+    private fun getFullUsername(account: Account): String {
+        if (account.isRemote()) {
+            return "@" + account.username
+        } else {
+            val localUsername = account.localUsername
+            val domain = accountManager.activeAccount!!.domain
```

Comment:
!! is safe here?

Reply:
Can you use the passed in `account: Account` instead of `activeAccount`? That way it's definitely non-null.

Reply:
No. `activeAccount` is an `AccountEntity`, which is why it has the `domain` field `account` does not.

Reply:
Yes I think it is safe.